Tools/jit/_llvm.py

```diff
@@ -32,12 +33,21 @@ async def wrapper(
     return wrapper
 
 
-_CORES = asyncio.BoundedSemaphore(os.cpu_count() or 1)
+_CORES_BY_LOOP: weakref.WeakKeyDictionary[
```

Comment:
Yeah, I've tried a couple of different things here, and this does work on my machine ™️  but I'm not sure that this is really the right approach here? Do we want to free this up between loops?